spyder/utils/misc.py

```diff
@@ -278,3 +279,15 @@ def getcwd_or_home():
         debug_print("WARNING: Current working directory was deleted, "
                     "falling back to home dirertory")
         return get_home_dir()
+
+
+def regexp_error_msg(pattern):
+    """
+    Returns None if the pattern is a valid regular expression or

```

Comment:
Imperative tense ("Return None"). Also, per PEP 257, docstrings must begin with a one-line summary, separated by a blank line from the rest of the docstring.